src/client/pages/Checkout/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -20,18 +20,14 @@ export const CheckoutDetails = () => {
   const { path: localePath } = useCurrentLocale()
   const { quoteCartId } = useQuoteCartIdFromUrl()
   const data = useQuoteCartData()
-  if (!data || data.loading) {
-    return <LoadingPage loading />
-  }
-  const onRetry = () => {
-    window.location.reload()
-    return false
-  }
 
-  if (data.error) {
-    console.error('Quote cart data error:', data.error.message, data.error)
+  if (data?.error) {
+    console.error('Quote cart data error: no data')
```

Comment:
Maybe we should remove the console.log?

Reply:
I think it helps to know what exactly failed since we display a really generic error everywhere.